style: apply /style-guide pass to models/automations (#2684)
wandb/docs: models/automations/registry-automation-tutorial.mdx

---
title: "Tutorial: Registry artifact alias automation"
description: Build an automation that runs a webhook when a Registry artifact gets a specific alias like "production".
keywords: ["artifact alias", "model promotion", "MLOps webhook", "alias regex", "model lifecycle"]
---
import TutorialDiagramRegistry from "/snippets/en/_includes/automations/tutorial-diagram-registry.mdx";
import TutorialNotebook from "/snippets/en/_includes/automations/tutorial-notebook.mdx";
import TutorialGoFurther from "/snippets/en/_includes/automations/tutorial-go-further.mdx";

> This tutorial walks you through building a **registry** automation triggered by artifact metadata: when an artifact in your registry gets a specific alias (for example, **production**), W&B sends a `POST` request to your webhook. Use this pattern to notify downstream systems (such as a deployment pipeline, paging service, or notification channel) whenever you promote a model to a known stage. This tutorial is intended for ML engineers and MLOps practitioners who manage model lifecycles in W&B Registry.

Reviewer: ```suggestion
This tutorial walks you through building a **registry** automation triggered by artifact metadata: when an artifact in your registry gets a specific alias (for example, **production**), W&B sends a `POST` request to your webhook. Use this pattern to notify downstream systems like deployment pipelines, paging services, or notification channels, whenever you promote a model to a known stage. This tutorial is intended for ML engineers and MLOps practitioners who manage model lifecycles in W&B Registry.
```

Author: Applied in 4992e00.

## Create a registry automation

> Follow these instructions to set up a registry-scoped automation. When an artifact in any collection in that registry gets a specific alias (for example, **production**), W&B sends a `POST` request to your webhook.

Reviewer: Nit: Whichever direction you go, align this with https://github.com/wandb/docs/pull/2684/changes#diff-ae3dac19c70f6d4851ac17870b1f1a097c99b57daa615000079f89efea9f150dR10.

Author: Rewrote line 23 to mirror `project-automation-tutorial.mdx` line 23: `Set up a registry-scoped automation so that when an artifact in any collection in the registry...` Parallel structure with the sibling tutorial. Applied in 4992e00.
